src/test/scala/Constants.scala

```diff
@@ -12,7 +12,10 @@ trait Constants {
   protected val TIMEOUT = 5 seconds
   protected val api = Constants.API
 
-  protected val user = "fanwashere" //REQUIRED: Edit this to your own username.
+  protected val user = "sukeshni" //REQUIRED: Edit this to your own username.
```

Comment:
FYI @fanwashere

It is not good to rewrite this line by each developer.
Use sys.env("xxxxx")

And write required environment variables to README.